README.md

```diff
@@ -14,27 +14,33 @@ In this project, we are setting up a "Hello world" repository with the use of on
 
 [Live Demo Link](http://127.0.0.1:5500/)
 
-
 ## Getting Started
 
 **This is an example of how you may give instructions on setting up your project locally.**
 **Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
 
-
 ### Prerequisites
-Make sure: 
+
+Make sure:
+
 - You have a <Windows/Linux/Mac> machine. State which OS is supported/which is not.
 - Good internet connection.
+
 ### Setup
+
 - Create a Github repository and name it "Hello-microverse""
 - Add a small description, a Readmefile and lisence(MIT)
 - After creating the Repo, clone the repo into your Pc and open the file into your editor.
 - Configurate Linters
```

Comment:
once linters are configured once on a repo, all other users don't need to install it again.